lib/bolt/project.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'pathname'
 require 'bolt/pal'
+require 'bolt/config'
```

Comment:
This circular dependency seems weird, but I don't think we can remove the project dependency from config and I'm not sure where else to put `system_path` that makes sense...

Reply:
`Bolt::Util`, maybe? `Bolt::Analytics` also uses the user path, so it could be nice to reuse it there as well.

Reply:
Yeah I noticed that too! If you have time to replace that in this PR I think that'd be great.

I think Util is more for Bolt-agnostic helper functions, and this feels Bolt-specific. I think Config is the right place for it, and in terms of ruby loading it's a nonissue, it just felt weird.